bin/get_experiment_info.sh

```diff
@@ -3,7 +3,6 @@
 set -e
 
 scriptDir=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
-source $scriptDir/db_scxa_common.sh
```

Comment:
Just checking don't you need `source common_routines.sh` here?

Reply:
It’s explained in the PR description above. However, this script isn’t using any imported functions.